explain/setup/self_managed/04_test_and_verify.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: 'Step 4: Test and Verify'
+install_track_title: 'Automated EXPLAIN: Setup (Self-Managed)'
+backlink_href: /docs/explain/setup
+backlink_title: 'Automated EXPLAIN: Setup'
+---
+
+import VerifyExplainPublic from '../_verify_explain_public.mdx'
+
+At this point, your pganalyze setup should be ready to collect EXPLAIN plans. Since
+you may not have any slow queries in your system right now, we can generate a
+synthetic slow query that we can use to confirm everything is working.
+
+The Postgres `pg_sleep` function, which just waits for the specified number of seconds,
+is perfect for this. Check your current `auto_explain.log_min_duration`:
+
+```sql
+SHOW auto_explain.log_min_duration;
+```
+
+And then run a `pg_sleep` with a value just over this (remember, the `log_min_duration`
+value will be shown in milliseconds, but `pg_sleep` takes an argument in seconds). E.g.,
+if your `auto_explain.log_min_duration` is 1000ms, you can run
+
+```sql
+SELECT pg_sleep(1.1);
```

Comment:
Should we use `--test-explain` instead here?

Reply:
Yes!